### PR TITLE
James/deep collection to sgov 156126489

### PR DESCRIPTION
**Pull request by jamesisaacs**

### CODE REVIEW

File: app/models/site_search.rb [hunk 1]
```diff
@@ -21,3 +20,4 @@ def domains_scope_options
   def populate_additional_results
   end
 end
+
```
Comment (MothOnMars) [inline app/models/site_search.rb]: looks like an extra line break snuck in

File: app/controllers/searches_controller.rb [hunk 2]
```diff
@@ -159,6 +159,7 @@ def search_consumer_urls
   end
 
   def docs_search_klass
+    @search_options[:affiliate].search_engine = 'SearchGov' if @search_options[:document_collection]&.too_deep_for_bing?
     return I14ySearch if gets_i14y_results?
```
Comment (MothOnMars) [inline app/controllers/searches_controller.rb]: Shouldn't this be:
```ruby
def docs_search_klass
  return I14ySearch if gets_i14y_results? || @search_options[:document_collection]&.too_deep_for_bing?
```
?

File: app/models/site_search.rb (hunk 1 above)
Comment (jamesisaacs) [reply app/models/site_search.rb]: Thanks.

File: app/controllers/searches_controller.rb (hunk 2 above)
Comment (jamesisaacs) [reply app/controllers/searches_controller.rb]: The search engine needed to be set in addition to sending I14ySearch as the search class.

**Comment by dawnpm** [comment]
@MothOnMars is this ready to go? thank you.

**Comment by MothOnMars** [comment]
LGTM 👍

**Pull request merged**